docs/sources/mimir/configure/about-versioning.md

```diff
@@ -96,6 +96,12 @@ The following features are currently experimental:
   - Early TSDB Head compaction to reduce in-memory series:
     - `-blocks-storage.tsdb.early-head-compaction-min-in-memory-series`
     - `-blocks-storage.tsdb.early-head-compaction-min-estimated-series-reduction-percentage`
+- Ingester client
+  - Per-ingester circuit breaking based on requests timing out or hitting per-instance limits
```

Comment:
[nit] Please add `-ingester.client.circuit-breaker-max-half-open-requests` to the list too.

Reply:
Changed the configuration settings in e9fc92026c936af1eb2d9d83aad333f1a31ddbda, all options are now listed here.